snykTask/src/__tests__/task-lib.test.ts

```diff
@@ -226,6 +226,41 @@ describe('getOptionsForSnykToHtml', () => {
   });
 });
 
+describe('generateSnykCodeResultsWithoutIssues sends console output to file', () => {
```

Comment:
Issue: The name of the test indicates that the function `generateSnykCodeResultsWithoutIssues()` would be tested, this is not the case. instead the test replicates what the function does in a way that doesn't test the actual code. I would say that this test doesn't help to ensure the quality of the code and can be removed. 

I will commit a basic test myself, which we can use to discuss on what else needs to be tested.